web/themes/custom/usagov/templates/includes/homepage-benefits-block.html.twig

```diff
@@ -0,0 +1,14 @@
+{% set lang = node.langcode.value %}
+
+{%  if (node.field_homepage_benefits_ref) %}
+  {% set benefits_url = drupal_url('node/' ~ node.field_homepage_benefits_ref.value[0].target_id ) %}
+    <div class="benefits-block">
+      <img src="/themes/custom/usagov/images/ICONS_beta_Benefits.svg" alt="Visit the benefit finder page" height="56px">
+      <div class="benefits-text">
+        <h2>{{ node.field_homepage_benefits_title.value }}</h2>
+        <p>{{ node.field_homepage_benefits_descr.value | striptags }}</p>
```

Comment:
We want to maintain the rich text from the field here (and we trust the source). So I think what we want is to remove the paragraph tag and change "striptags" to "raw": 

```{{ node.field_homepage_benefits_descr.value | raw }}```

(I noticed this because I'd inadvertently included a non-breaking space in my benefits callout text, and I saw "&nbsp;" on the page.)

Reply:
removed the p tag and changed striptags to raw